documentation/auto-discovery/topics/advanced-discovery-topics.rst

```diff
@@ -158,3 +152,30 @@ By default, the Docker software does not perform log rotation for the "local/JSO
         "max-file": "3"
       }
     }
+
+==============================
+HTTP/HTTPS Proxy configuration
+==============================
+
+Docker daemon
+-------------
+Docker publishes `detailed instructions <https://docs.docker.com/engine/daemon/proxy/>`_ on how to configure the docker daemon to use a proxy server.
+
+User environment
+----------------
+Ensure that the user environment is configured. This is by ensuring the following environment variables have the correct values.
+
+- HTTP_PROXY
+- http_proxy
+- HTTPS_PROXY
+- https_proxy
+- NO_PROXY
+- no_proxy
+
+Proxy software
+--------------
+The proxy software needs to allow access to three services.
+
+1. Your Hyperview instance hostname/URL. This URL is created when you activate your subscription.
+2. Data collector installer. The data collector installer is hosted on Microsoft Azure Blob Storage. You can obtain the address from your instance by navigating to *Discoveries -> Data Collectors -> Download Data Collector*.
+3. Data collector docker containers. These are hosted in a private Microsoft Azure container registry. You can obtain the registry address from the docker-compose file bundled with the installer. Please note that each registry has two addresses, an API/REST address and a data address. Usually allowing the API/REST endpoint address,<registry name>.azurecr.io, and <proxy config wildcard>.data.azurecr.io is adequate.
```

Comment:
Missing a space after "...endpoint address,..."

Reply:
Or replace the comma with a space.

Reply:
Thanks! Updated.